Assets/_BForBoss/_Entities/Scripts/PlaceholderObstacleDamageBehaviour.cs

```diff
@@ -0,0 +1,16 @@
+using UnityEngine;
+
+namespace Perigon.Entities
+{
+    public class PlaceholderObstacleDamageBehaviour : MonoBehaviour
+    {
+        private void OnCollisionEnter(Collision other)
+        {
+            var lifeCycle = other.gameObject.GetComponent<LifeCycleBehaviour>();
+            if (lifeCycle != null)
+            {
+                lifeCycle.Damage(50);
```

Comment:
I know its a place holder, but we could make 50 into a variable that we can tweak (i.e. `[SerializeField] private int _damageAmount`) So that we can play with values for different obstacles (for whatever reason we might need to test in the future)

Reply:
sure